docker-compose.yml

```diff
@@ -8,6 +8,6 @@ services:
       - '5438:5432'
```

Comment:
Should the postgres version be 10.x to parity the Aurora RDS PG version? Or does that not matter?

Reply:
Good question but... https://aws.amazon.com/about-aws/whats-new/2022/08/amazon-aurora-serverless-v1-postgresql-11-in-place-upgrade-postgresql-10/
should be able to use 11 very shortly, probably next CDK/SST update